packages/frontend/src/components/MultiSelect/index.tsx

```diff
@@ -54,7 +55,10 @@ function MultiSelect(props: MultiSelectProps): React.ReactElement {
       }) => (
         <FormControl isInvalid={!!error}>
           {label && (
-            <FormLabel isRequired={required} description={description}>
+            <FormLabel
+              isRequired={required}
+              description={description && <Markdown>{description}</Markdown>}
```

Comment:
```suggestion
              description={description && <Markdown linkTarget='_blank'>{description}</Markdown>}
```
to open all links in new window. There shouldn't be a need to open in the same window for descriptions right.